src/main/java/org/killbill/billing/plugin/adyen/client/payment/converter/impl/NullObjectConverter.java

```diff
@@ -25,7 +25,7 @@
 public class NullObjectConverter implements PaymentInfoConverter {
 
     @Override
-    public PaymentType getPaymentType() {
+    public boolean supportsPaymentType(PaymentType type) {
         throw new UnsupportedOperationException("method is not supported!");
```

Comment:
Well, the `UnsupportedOperationException` means this operation is not supported at all, not just for some values of `type` … but seeing the `type` might help debugging --> I'll add it :smiley: